content/en/blog/recap-2022/index.md

```diff
@@ -0,0 +1,136 @@
+
+- [Falco 0.31.0 a.k.a. "the Gyrfalcon"](https://falco.org/blog/falco-0-31-0/) released in January 2022.  Why did we call it the Gyrfalcons release? Well, the Gyrfalcons are the largest of the falcon species, just like this version of Falco, which had the biggest changelog ever released.The falco and libs repositories counted 30+ individual contributors, 130+ pull requests, and 360+ commits 🤯. 
+
+- [Falco 0.32.0](https://falco.org/blog/falco-0-32-0/) release in June 2022. This new release came with a ton of inner rework. Here is the foremost important change: Lua is no more a dependency of Falco! Basically, the Falco rule loader was rewritten in C++, to achieve better performance. Moreover, the entire rule engine has been rewritten too. This reduced the workarounds in Falco, and is now fully using libsinsp-provided filter parsers and compiler. Moreover, a new --list-syscall-events CLI option is now available. 
+
+- [Falco 0.33.0 a.k.a. "the pumpkin release 🎃"](https://falco.org/blog/falco-0-33-0/)  made available October 2022.  For v0.33.0 the community focused on addressing the following updates & changes:
+  - Libs now allow individual selection of which syscalls to collect during live captures, which helps Falco improve performance and reduce dropped events
+  - Introduced the Kernel Crawler, a new tool that automatically identifies the most up to date kernel versions supported by popular distros
+  - Syscall kernel ring-buffer size is now customizable for your environment needs
+  - Mitigations for libsinsp’s Kubernetes metadata client to address recent issues that caused Falco to crash
+  - Support for multiple simultaneous event sources, which means that you can now run multiple event sources in the same Falco instance
+  - Added minikube as a supported platform in the driver loader and included it in our driver build matrix
+  - Rule alert rate limiter is now optional and disabled at default
+  - Support for two new syscalls and many improvements to the default Falco security ruleset
+
+### Falcosidekick & UI
+[Falcosidekick](https://github.com/falcosecurity/falcosidekick) is a little daemon that extends a number of possible outputs, and since its creation this little guy has evolved in many amazing ways. A big thank you to Thomas Labarussias for driving the development of Falcosidekick and the UI. Here is how the project evolved in 2022: 
+
+- Falcosidekick saw a major update in June of 2022, delivering four new outputs and enhancing existing ones too! Not only did we see updates in outputs, but the community delivered the first version of the Falcosidekick UI. The UI offered these new features: Redis DB for long term storage of events, API for counting and searching for events, and lastly filters to keep and share query strings. For a full detailed account of this update, see the blog Falcosidekick 2.25.0 and Falcosidekick 2.0.0. 
```

Comment:
For a full detailed account of this update, see the blog [Falcosidekick 2.25.0 and Falcosidekick 2.0.0. ](https://falco.org/blog/falcosidekick-2-25-0-falco-2-0-0/)

Reply:
```suggestion
- Falcosidekick saw a major update in June of 2022, delivering four new outputs and enhancing existing ones too! Not only did we see updates in outputs, but the community delivered the first version of the Falcosidekick UI. The UI offered these new features: Redis DB for long term storage of events, API for counting and searching for events, and lastly filters to keep and share query strings. For a full detailed account of this update, see the blog [Falcosidekick 2.25.0 and Falcosidekick 2.0.0.](https://falco.org/blog/falcosidekick-2-25-0-falco-2-0-0/).
```